### PR TITLE
UI changes

### PR DESCRIPTION
**Pull request by srishtigautam21**
Added favicon and little bit of UI changes

### CODE REVIEW

**Review by Kavita539 — APPROVED** [review]
- code looks neat and clean
- Folder structure is good

File: css/landingPage.css [hunk 1]
```diff
@@ -1,34 +1,45 @@
...
+  color: var(--COLOR-PRIMARY);
+}
+.github-text-color:hover {
+  color: var(--COLOR-WHITE);
+}
+.button-hover {
+  color: var(--COLOR-WHITE);
+}
+.landing-page-container {
+  /* margin: 9rem 10rem 5rem 21.5rem; */
```
Comment (Megha-Joshi) [inline css/landingPage.css]: Remove the commented code, if not required.

**Review by Megha-Joshi — APPROVED** [review]
- Code looks good.
- Nice use of variables.
- Good to merge

File: css/landingPage.css (hunk 1 above)
Comment (srishtigautam21) [reply css/landingPage.css]: done

**Pull request merged**